app/logic/fileHandler.py

```diff
@@ -61,10 +75,13 @@ def deleteFile(self, fileId):
         """
         Deletes attachmant from the app/static/files/eventattachments/ or courseattachments/ directory
         """
-        try:
-            file = AttachmentUpload.get_by_id(fileId)
+        file = AttachmentUpload.get_by_id(fileId)
+        filename = file.fileName
+        file.delete_instance()
+
+        # checks if there are other instances with the same filename in the AttachmentUpload table
+        if AttachmentUpload.select().where(AttachmentUpload.fileName == file.fileName).exists():
+            pass
+        else:
```

Comment:
Instead of having nothing in the positive case and the actual functionality in the `else`, just change the conditional so that the `else` is unnecessary and the code is in the `if` block.